### PR TITLE
TeamItem UI Updated.

### PR DESCRIPTION
**Pull request by divyanshu-rawat**
Draft changes to update TeamItem UI to look similar to the one attached in the #40.

@domyen So, I have created this draft PR to start working in #40 .

* In this I have updated the `TeamItem`, to look like the one you have attached in the issue description. 👍 .

Fell free to have a look in `netlify` preview generated, Let me know your thoughts in it, and I will update it accordingly.

### CODE REVIEW

File: src/components/screens/TeamScreen/Contributors.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+`;
+
+const StyledLink = styled(Link)`
+  margin-top: 12px;
+`;
+
+const PureContributors = ({ contributors }) => (
+  <Wrapper>
+    <section>
+      <Heading>+{parseInt(contributors.length / 3, 10)} Contributors</Heading>
```
Comment (divyanshu-rawat) [inline src/components/screens/TeamScreen/Contributors.js]: I think I can just send contributors from contributors function instead of doing it here. 😅

Comment (kylesuss) [reply src/components/screens/TeamScreen/Contributors.js]: Yeah good callout. I just did a quick look at the GitHub rest API and it looks like you can do something like this:

`https://api.github.com/repos/storybookjs/frontpage/contributors?per_page=2`

So maybe you can update the `contributorsUrl` below to include that query param with the appropriate value and then you won't have to worry at all about having to chop up the array?

**Comment by divyanshu-rawat** [comment]
@kylesuss updated feel free to have a look 👍

**Comment by kylesuss** [comment]
Hey @divyanshu-rawat -- thanks for taking this on & submitting the PR 😄 

Before I review, I want to understand where you are trying to get this PR -- is the end goal to match the design that was presented in this issue?

https://github.com/storybookjs/frontpage/issues/40#issuecomment-497893640

If so, I am seeing some differences so I'm curious about your thought process and what your goal with this PR may be. Any thoughts on the matter @divyanshu-rawat ?

For example, this section has some differences:

Screen Shot 2019-12-12 at 9 43 58 AM

**Comment by divyanshu-rawat** [comment]
got it @kylesuss at this moment, I am done with adding contributors section, though I made some improvements in the steering committee as per design added by @domyen , and we will leave the maintainers section to be curated by @shilman as per @domyen :).

Now, If you want I can also finish the `steering committee` and `contributors` and @shilman is working on `maintainers section`.

**Comment by kylesuss** [comment]
@divyanshu-rawat great. Thanks for the explanation. I just created a new branch on this repo, `new-team-page` that we can use to capture all of these changes. I am hesitant to merge anything into `master` until it is ready to go since master gets automatically deployed!

I am going to merge this PR into that branch and push a few changes up there as well. I have some time today so I can tackle the remaining steering committee & contributors updates, unless of course you wanted to do it? I am happy to help but don't want to steal your work if you really wanted it -- just trying to help move this along 😄

**Comment by divyanshu-rawat** [comment]
Cool :)  @kylesuss , Ya I have also face similar problem in my personal projects as master gets auto-deployed via my `Heroku` pipeline, Nevertheless I will update the PR by adding `steering committee` then. 🎉

**Comment by kylesuss** [comment]
Sounds good @divyanshu-rawat -- all yours. I won't merge this PR just yet then. Try to get as close as you can to the design from here:

https://github.com/storybookjs/frontpage/issues/40#issuecomment-497893640

...and I'll be available when you're done and need a review 👌

**Comment by divyanshu-rawat** [comment]
> Sounds good @divyanshu-rawat -- all yours. I won't merge this PR just yet then. Try to get as close as you can to the design from here:
> 
> [#40 (comment)](https://github.com/storybookjs/frontpage/issues/40#issuecomment-497893640)
> 
> ...and I'll be available when you're done and need a review 👌

Sure 👍

**Comment by divyanshu-rawat** [comment]
@kylesuss I think it is now ready for review 🚀.
Feel free to have a look 👍

**Comment by kylesuss** [comment]
@divyanshu-rawat thanks for the heads up. I will review once I get a spare block of time this week 🙏

**Comment by divyanshu-rawat** [comment]
It was great working on the issue :)

**Pull request merged**